### DISCUSSION TITLE
哔哩安装指南

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#52] [通知]
如果想安装哔哩，请打开 [Release](https://github.com/Richasy/Bili.Uwp/releases) 页面，找到最新版本，并选择适用于当前系统的安装包下载。

然后打开 [系统设置](ms-settings:developers)，打开 `开发者模式` ，并等待系统安装一些必要的扩展项。

在应用压缩包下载完成后，解压压缩包，并在管理员模式下，使用 PowerShell 运行解压后的 `install.ps1` 脚本，根据提示进行安装。

**Watch** 项目，以获取应用的更新动态。

关于如何一步步地使用侧加载 (Sideload) 方式安装 UWP 应用及订阅应用更新，请参见 [下载并安装哔哩的详细说明](https://github.com/Richasy/Bili.Uwp/blob/main/how_to_install.md) 。

**Comment by Andy0212589** [comment]
无法验证此应用包的发布者证书。请与系统管理员或应用开发人员联系，以获取具有验证证书的新应用包。必须验证应用包中签名的根证书和所有直接证书(0x800B010A)怎么办
